cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorTestUtils.java

```diff
@@ -42,16 +42,23 @@ private AnomalyDetectorTestUtils() {
   /**
    * Create history load for broker.
    * @param metricValueById A set of metrics with corresponding load value.
+   * @param metricVarianceById A set of metrics with corresponding load variance.
    * @param numWindows  Number of windows to populate.
    * @param broker The subject broker.
    * @return The load for the broker.
    */
-  public static Map<BrokerEntity, ValuesAndExtrapolations> createHistory(Map<Short, Double> metricValueById, int numWindows, BrokerEntity broker) {
+  public static Map<BrokerEntity, ValuesAndExtrapolations> createHistory(Map<Short, Double> metricValueById,
+                                                                         Map<Short, Double> metricVarianceById,
+                                                                         int numWindows,
+                                                                         BrokerEntity broker) {
+    Random random = new Random();
     Map<Short, MetricValues> valuesByMetricId = new HashMap<>();
     for (Map.Entry<Short, Double> entry : metricValueById.entrySet()) {
       MetricValues historicalMetricValues = new MetricValues(numWindows);
       double[] values = new double[numWindows];
-      Arrays.fill(values, entry.getValue());
+      for (int i = 0; i < numWindows; i++) {
+        values[i] = entry.getValue() + (random.nextInt(201) - 100) * metricVarianceById.get(entry.getKey()) / 100;
```

Comment:
It is not clear what these hardcoded magic numbers refer to or the purpose of this operation.
Can we clarify both?